tests/schemas/invalid/test_invalid_mismatching_answer_label_and_value.json

```diff
@@ -0,0 +1,207 @@
+{
```

Comment:
Can we reduce this schema and get rid of all the things we don't need like routing rules etc. All we need is a 1 question with answer options that has a mismatch. Worth adding a placeholder one too.

Reply:
I've simplified the schema as best as I could with the addition of a placeholder question.